java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2951,7 +2940,7 @@ public <ArgType> T zintercard(@NonNull ArgType[] keys, long limit) {
      * , and stores the result in <code>destination</code>. If <code>destination</code> already
      * exists, it is overwritten. Otherwise, a new sorted set will be created.<br>
      * To perform a <code>zinterstore</code> operation while specifying aggregation settings, use
-     * {@link #zinterstore(String, KeysOrWeightedKeys, Aggregate)}
+     * {@link #zinterstore(Object, KeysOrWeightedKeys, Aggregate)}.
```

Comment:
Why Object? Is it because this could be String or GlideString

Reply:
I don't know why, but it can't resolve the link if I put there `ArgType`.
Strange, but it can be resolved from a function which has `ArgType` as a type parameter. Weird think.